plutus-core/plutus-core/src/PlutusCore/Builtin/Polymorphism.hs

```diff
@@ -214,7 +215,8 @@ type family AllElaboratedArgs constr x where
 -- built-in type.
 type AllBuiltinArgs
         :: forall a. (GHC.Type -> GHC.Type) -> (GHC.Type -> GHC.Constraint) -> a -> GHC.Constraint
-type AllBuiltinArgs uni constr x = AllElaboratedArgs constr (ElaborateBuiltin uni x)
+class    AllElaboratedArgs constr (ElaborateBuiltin uni x) => AllBuiltinArgs uni constr x
+instance AllElaboratedArgs constr (ElaborateBuiltin uni x) => AllBuiltinArgs uni constr x
```

Comment:
There was no reason to make it a class synonym before, but now it's convenient, since we partially apply it in the `TestAllBuiltinsSatisfy` instances below.